@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf

```diff
@@ -42,8 +42,8 @@ if !(_heli getVariable ["fza_ah64_aircraftInitialised", false]) then {
     _heli setVariable ["fza_ah64_pfz_count", 0, true];
     _heli setVariable ["fza_ah64_curwpnum", 0, true];
     _heli setVariable ["fza_ah64_waypointdata", [getPos _heli], true];
-    _heli setVariable ["fza_ah64_sight_plt", 0, true];
-    _heli setVariable ["fza_ah64_sight_cpg", 0, true];
+    _heli setVariable ["fza_ah64_sight_plt", 2, true];
```

Comment:
Are you sure you want the default sight select in the pilot's seat to be TADS? I would recommend HMD for both.